metadata.json

```diff
@@ -31,7 +31,8 @@
       "operatingsystemrelease": [
         "12.04",
         "14.04",
-        "16.04"
+        "16.04",
+	"18.04"
```

Comment:
Odd indenting here due to mixing of tabs and spaces

Reply:
Looks better now.